manifest-production.yml

```diff
@@ -0,0 +1,8 @@
+---
+inherit: manifest-base.yml
+
+domain: dashboard.gov.au
```

Comment:
Shouldn't this domain be `smart-cities.dashboard.gov.au` ?

Reply:
One would think so, but no, it actually is the parent domain, where the app name ends up sitting as a prefix underneath.

Confusing right? :)

Reply:
Ah I see, thanks. Maybe drop in a quick comment to that effect in the file?

Reply:
Done